beacon-chain/core/transition/transition_no_verify_sig.go

```diff
@@ -243,7 +243,7 @@ func ProcessOperationsNoVerifyAttsSigs(
 		if err != nil {
 			return nil, err
 		}
-	case version.Altair, version.Bellatrix:
+	case version.Altair, version.Bellatrix, version.Capella:
```

Comment:
```suggestion
	case version.Altair, version.Bellatrix
```
Better to leave transition changes out of this PR.